bin/pygrb/pycbc_pygrb_plot_injs_results

```diff
@@ -51,451 +42,356 @@ __program__ = "pycbc_pygrb_plot_injs_results"
-MF = np.argsort(g_ifar['stat'])
-# Statistics: found-missed
-FM = MF[::-1]
-
-logging.info("%d found injections analysed.", len(found_injs))
-
-# Post-processing of injections ends here
-
-# ==========
-# Make plots
-# ==========
-
-# Plot results of injection campaign with found ones on top of missed ones
-# (found-missed) or vice-versa (missed-found)
-
-# Info for site-specific plots
+# Load indices containing found and missed injections
+found = f['found_after_vetoes/injection_index'][:]
+missed = f['missed/after_vetoes'][:]
+
+# Load inclusive and exclusive IFARs
+if opts.far_type == 'inclusive':
+    ifar_found = f['found_after_vetoes/ifar'][:]
+    far_title = 'Inclusive'
+elif opts.far_type == 'exclusive':
+    ifar_found = f['found_after_vetoes/ifar_exc'][:]
+    far_title = 'Exclusive'
+
+# Load ifos that are available in the HDF file
+if 'ifos' in f.attrs:
+    sites = f.attrs['ifos'].split(' ')
+else:
+    logging.warning("Ifos not found in input file, assuming H-L")
+    sites = ['H1', 'L1']
+
+# Load data
+x_qty = opts.x_variable
+y_qty = opts.y_variable
+inj_data = load_data(opts.injection_file, [x_qty, y_qty], opts, sites)
+
+# Specify indices for found and missed injections
+fxvals = inj_data[x_qty][found]
+mxvals = inj_data[x_qty][missed] 
+fyvals = inj_data[y_qty][found]
+myvals = inj_data[y_qty][missed]
+
+logging.info('Data loaded.')
+
 sitename = {'G1':'GEO', 'H1':'Hanford', 'L1':'Livingston', 'V1':'Virgo',
             'K1':'KAGRA'}
+labels={'mchirp': "Chirp Mass (solar masses)",
+        'mtotal': "Total mass (solar masses)",
+        'q': "Mass ratio",
+        'distance': "Distance (Mpc)",
+        'eff_site_dist': "%s effective distance (Mpc)" % sitename.get(opts.ifo),
+        'eff_dist': "Inverse sum of effective distances (Mpc)",
+        'end_time': "Time since %d (s)" % grb_time,
+        'coa_phase': "Phase of complex SNR (radians)",
+        'latitude': "Latitude (radians)",
+        'longitude': "Longitude (radians)",
+        'incl': "Inclination (iota)",
+        'abs_incl': 'Magnitude of inclination (|iota|)',
+        'cos_incl': "cos(iota)",
+        'cos_abs_incl': "cos(|iota|)",
+        'mass1': "Mass of 1st binary component (solar masses)",
+        'mass2': "Mass of 2nd binary component (solar masses)",
+        'polarization': "Polarization phase (radians)",
+        'spin1': "Spin on 1st binary component",
+        'spin1x': "Spin x-component of 1st binary component",
+        'spin1y': "Spin y-component of 1st binary component",
+        'spin1z': "Spin z-component of 1st binary component",
+        'spin2': "Spin on 2nd binary component",
+        'spin2x': "Spin x-component of 2nd binary component",
+        'spin2y': "Spin y-component of 2nd binary component",
+        'spin2z': "Spin z-component of 2nd binary component"}
+
+logging.info('Plotting...')
+if opts.missed_on_top:
+  fig_title = 'Missed and Found Injections'
+else:
+  fig_title = 'Found and Missed Injections'
```

Comment:
`fig_title = 'Missed and Found Injections' if opts.missed_on_top else 'Found and Missed Injections'`